01-shapefile-attributes.Rmd

```diff
@@ -36,33 +36,35 @@ values.
 ###Goals / Objectives
 After completing this activity, you will:
 
- * Be able to query shapefile attributes
- * Be able to subset shapefiles based on specific attributes
+ * Be able to query shapefile attributes.
+ * Be able to subset shapefiles using specific attribute values..
+ * Know how to plot a shapefile, colored by unique attribute values.
  
-###What you'll need
-
-You will need the most current version of R or R studio loaded on your computer 
-to complete this lesson.
-
-###R Libraries to Install:
+###Things You'll Need To Complete This Lesson
+You will need the most current version of `R` and, preferably, RStudio loaded on
+your computer to complete this lesson.
 
+####R Libraries to Install:
 
-* **raster:** `install.packages("raster")`
-* **sp:** `install.packages("sp")`
 * **rgdal:** `install.packages("rgdal")`
-
+* **sp:** `install.packages("sp")`
```

Comment:
Even though many people may just stumble on this one page?  What cut off for number of lessons in a series does it have to be included to be in a lesson?  I'm just thinking in the other series there is also some lessons with more packages than others.